installer/resources/lambda_rule_engine/files/rule_engine_cloudwatch_aws_rules.json

```diff
@@ -232,10 +232,10 @@
     "ruleExecutable": "",
     "ruleRestUrl": "",
     "ruleType": "Manage Rule",
-    "ruleArn": "arn:aws:events:us-east-1:***REMOVED***:rule/aws_classic_elb_should_not_be_publicly_accessible",
+    "ruleArn": "arn:aws:events:us-east-1:***REMOVED***:rule/aws_classic_elb_should_not_be_publicly_accessibleaws_classic_elb_should_not_be_publicly_accessible",
```

Comment:
Check if this name is correct, it seems name is repeated

Reply:
Done